core/commonMain/src/kotlinx/serialization/modules/SerializersModule.kt

```diff
@@ -28,7 +29,21 @@ public sealed class SerializersModule {
      * This method is used in context-sensitive operations on a property marked with [Contextual] by a [ContextualSerializer]
      */
     @ExperimentalSerializationApi
-    public abstract fun <T : Any> getContextual(kclass: KClass<T>): KSerializer<T>?
+    @Suppress("INVISIBLE_REFERENCE", "INVISIBLE_MEMBER")
+    @Deprecated(
+        "Deprecated in favor of overload with default parameter",
+        ReplaceWith("getContextual(kclass)"),
+        DeprecationLevel.ERROR
+    )
+    @LowPriorityInOverloadResolution
+    public fun <T : Any> getContextual(kclass: KClass<T>): KSerializer<T>? =
```

Comment:
Why not making it HIDDEN straight away? Then `LowPriorityInOverloadResolution` will be redundant as well

Reply:
I think it's possible